### DISCUSSION TITLE
OS field lowercase

### DISCUSSION THREAD

**Discussion by OlegGulevskyy** [goreleaser/goreleaser#3981] [Ideas, Issue Triage, and General discussions]
Hello, hope you are well!
Is it possible to make sure that published package named in lowercase, instead of capitalised? I am using a default setting, which produces artifact like so:
```
foo_Darwin_arm64.tar.gz
```
Is it possible to ensure that `Darwin` is lowercases? When I publish the package to NPM and try to install it, it appears that `npm` is searching for `foo_darwin_arm64.tar.gz` file, so it fails with 404

Sorry if I am missing an elephant in the room

**Comment by OlegGulevskyy** [comment]
Well, actual binaries when built seem to have lowercased OS names, but when published to Assets of Github packages, they are uppercased
<img width="410" alt="image" src="https://user-images.githubusercontent.com/43781031/236331565-041f20f3-2f5d-4fb7-a0c1-73609590b832.png">

**Comment by caarlos0** [comment]
Check the docs for the archive name template, you can make a lot with it 👌

**Reply by OlegGulevskyy** [reply]
> Check the docs for the archive name template, you can make a lot with it 👌

my bad, sorry for silly question
figured it out - had to remove `title` property next to `.Os`
